lib/task_helpers/exports.rb

```diff
@@ -13,6 +13,15 @@ def self.parse_options
         opt :all, 'Export read-only objects', :type => :boolean, :default => false
       end
 
+      def ensure_absolute_path(dir)
```

Comment:
Seems the method is not needed... You can just inline this code.

Reply:
Ah, right you are!  I'll simplify it.